robottelo/hosts.py

```diff
@@ -446,15 +462,57 @@ def check_services(self):
             if error_msg in line:
                 return line.replace(error_msg, '').strip()
 
-    def install(self, **cmd_kwargs):
+    def install(self, installer_obj=None, cmd_args=None, cmd_kwargs=None):
         """General purpose installer"""
-        command_args = {'scenario': self.__class__.__name__.lower()}
-        command_args.update(cmd_kwargs)
-        command_args = ' '.join(
-            [f'--{key.replace("_", "-")} {value}' for key, value in command_args.items()]
+        if not installer_obj:
+            command_opts = {'scenario': self.__class__.__name__.lower()}
+            command_opts.update(cmd_kwargs)
+            installer_obj = InstallerCommand(*cmd_args ** command_opts)
```

Comment:
`InstallerCommand(*cmd_args ** command_opts)` missing ',' or I dont know this pattern

Reply:
ooohhhh good catch